docs/references/helmrelease-custom-resource.md

```diff
@@ -111,6 +111,108 @@ OK
 > either need to port forward before making the request or put something
 > in front of it to serve as a gatekeeper.
 
+## Extending the supported Helm repository protocols
+
+By default, the Helm operator is able to pull charts from repositories
+using HTTP/S. It is however possible to extend the supported protocols
+by making use of a [Helm downloader plugin](https://helm.sh/docs/topics/plugins/#downloader-plugins),
+this allows you for example to use charts hosted on [Amazon S3](https://github.com/hayorov/helm-gcs)
+or [Google Cloud Storage](https://github.com/hayorov/helm-gcs).
+
+> **Note:** the operator only offers support for _downloader plugins_,
+> other plugins will not be recognized nor used.
+
+**Plugin folder paths per Helm version:**
+
+| Version | Plugins | Config
+|--------|---------|---
+| Helm 2 | `/var/fluxd/helm/cache/plugins` | `/var/fluxd/helm/plugins`
+| Helm 3 | `/root/.cache/helm/plugins` | `/root/.local/share/helm/plugins`
+
+### Install a Helm downloader plugin
+
+The easiest way to install a plugin so that it becomes accessible to
+the Helm operator to use an [init container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)
+and one of the available `helm` binaries in the operator's image and
+a volume mount. For the Helm chart of the operator, [see the
+documentation](https://github.com/fluxcd/helm-operator/tree/master/chart/helm-operator#use-helm-downloader-plugins).
+
+#### Using an init container
+
+Create a volume entry of [type `emptyDir`](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir)
+to the deployment of your Helm operator, this is where the plugins will
+be stored for the lifetime duration of the pod.
+   
+```yaml
+spec:
+  volumes:
+  - name: helm-plugins-cache
+    emptyDir: {}
+```
+   
+Add a new init container that utilizes the same image as the operator's
+container and makes use of the earlier mentioned volume with correct
+volume mounts for the Helm version your are making use of.
+The available `helm2` and `helm3` binaries can then be used to install
+the plugin.
+   
+```yaml
+spec:
+  initContainers:
+  - name: helm-3-downloader-plugin
+    image: docker.io/fluxcd/helm-operator:<tag>
+    imagePullPolicy: IfNotPresent
+    command: ['sh', '-c', 'helm3 plugin install <plugin> --version <version>']
```

Comment:
The placeholders like \<plugin\>, \<version\> etc.. were not immediately obvious to me, wonder if some bold here might help?